test/order_status_schema.test.js

```diff
@@ -0,0 +1,37 @@
+const OrderStatus = require('../src/dronuts-backend/order_status_schema');
+const expect = require('chai').expect;
+
+describe('Onder Status', function() {
+    it('order status should always contain an order id', function(done) {
+        m = new OrderStatus();
+        m.validate(function(err) {
+            expect(err.errors.orderId).to.exist;
+            done();
+        });
+    });
+
+    it('order status should always contain a customer id', function(done) {
+        m = new OrderStatus();
+        m.validate(function(err) {
+            expect(err.errors.droneId).to.exist;
```

Comment:
I think this should be customerId

Reply:
yeah good catch, the description should be “always contain a drone id"